client/my-sites/domains/domain-management/settings/cards/name-servers-card.tsx

```diff
@@ -85,11 +85,23 @@ const NameServersCard = ( {
 			return false;
 		}
 
-		return nameservers.every( ( nameserver ) => {
+		return nameservers.some( ( nameserver ) => {
```

Comment:
This would hide the warning for someone with one Cloudflare nameserver mixed with other non-Cloudflare ones. Is that okay?

Reply:
I think we should keep this as `every` due to the reason Ramy mentioned 🤔 it should be ok to use only Cloudflare name servers, but using different mixed name servers usually lead to problems

Reply:
Also, another interesting side-effect of this change is that if you input a name server and then erase it, the warning also disappears:

https://github.com/user-attachments/assets/9896151b-448b-4543-9518-80fa82e7021e

Reply:
Great suggestions, fixing! :)